dashboard/lib/tasks/seed.rake

```diff
@@ -106,7 +106,15 @@ namespace :seed do
     interactive-games-animations-2023
     focus-on-creativity3-2023
     focus-on-coding3-2023
+    csd1-2024
+    csd2-2024
     csd3-2024
+    csd4-2024
+    csd5-2024
+    csd6a-2024
+    csd6b-2024
+    csd7-2024
+    csd-post-survey-2024
     interactive-games-animations-2024
     focus-on-creativity3-2024
     focus-on-coding3-2024
```

Comment:
I am guessing these were added since so much was from before 2021... perhaps we can use one of these `focus-on-codings` instead?

Reply:
I found a way to use allthethings instead. Really great catch though!